### PR TITLE
babel-env instead of babel-presets

### PR DESCRIPTION
**Pull request by arturparkhisenko**
for #4

### CODE REVIEW

File: api/webpack.config.babel.js [hunk 1]
```diff
@@ -18,11 +18,26 @@ module.exports = {
...
+        rules: [
             {
                 test: /\.js$/,
-                loader: 'babel-loader',
                 exclude: path.resolve(__dirname, 'node_modules'),
+                use: [{
+                    loader: 'babel-loader',
+                    options: {
+                        presets: [
+                            ['env', {
```
Comment (Kmaschta) [inline api/webpack.config.babel.js]: Can you configure babel in the `.babelrc` file ?
`babel-loader` automatically read this file

**Comment by Kmaschta** [comment]
It will be easier for you if you make a separated PR for the API and another one for the admin.

File: api/webpack.config.babel.js (hunk 1 above)
Comment (arturparkhisenko) [reply api/webpack.config.babel.js]: @Kmaschta yes, thanks)

**Comment by arturparkhisenko** [comment]
@Kmaschta I'll close and open with different PRs.

**Pull request closed without merging**